### DISCUSSION TITLE
Integration with clan/team plugins

### DISCUSSION THREAD

**Discussion by Noxillio** [GriefPrevention/GriefPrevention#1847] [Ideas]
Integration with clan and team plugins such as UltimateClans and Better Teams. This would allow clan/team members the ability to access builds/chests, and to operate Redstone circuits.

**Comment by Jikoo** [comment]
It's possible for an addon to do this, though I don't know of one that does. GP allows custom data in trust lists, which can then be interpreted in the ClaimPermissionCheckEvent. Base GP only parses permission nodes in brackets and UUIDs, everything else is ignored. The group plugins in question could add a GP hook adding this capacity.
